tools/depends/target/pythonmodule-crossenv/Makefile

```diff
@@ -0,0 +1,72 @@
+include ../../Makefile.include
+DEPS= ../../Makefile.include Makefile
+
+# lib name, version
+LIBNAME=crossenv
+VERSION=1.0
+BASE_URL=https://github.com/benfogle/$(LIBNAME)/archive
+ARCHIVE=v$(VERSION).tar.gz
+
+# provide any platform specific env variables to be available to the cross env
+ifeq ($(OS),android)
+  LDSHAREDCC=--env=LDSHARED:="$(CC) -shared $(PYTHON_MODULELDFLAGS)"
+  SDKROOT=$(NDKROOT)/sysroot
+endif
+ifeq (darwin, $(findstring darwin, $(HOST)))
+  ifeq ($(OS),darwin_embedded)
+    CROSS_CC_FLAGS=-arch $(CPU) --sysroot=$(SDKROOT)
+  endif
+  ZLIBROOT=--env=ZLIB_ROOT:="$(SDKROOT)/usr"
+  ifeq ($(TARGET_PLATFORM),appletvos)
+    # -fembedbitcode means we cannot do any undefined things like -undefined, -U<symbol>
+    # At linktime, have to supply static link flags
+    LDSHAREDCC=--env=LDSHARED:="$(CC) $(PYTHON_MODULELDFLAGS) $(LDFLAGS)"
+  else
+    LDSHAREDCC=--env=LDSHARED:="$(CC) -bundle -undefined dynamic_lookup $(LDFLAGS)"
+  endif
+endif
+
+ifneq ($(SDKROOT),)
+  SYSROOTENV=--sysroot $(SDKROOT)
+  PKGCONFIGSYSROOT=--env=PKG_CONFIG_SYSROOT_DIR:=$(SDKROOT)
+endif
+
+all: .installed-$(PLATFORM)
+
+$(TARBALLS_LOCATION)/$(ARCHIVE):
+	cd $(TARBALLS_LOCATION); $(RETRIEVE_TOOL) $(RETRIEVE_TOOL_FLAGS) $(BASE_URL)/$(ARCHIVE)
+
+$(PLATFORM): $(TARBALLS_LOCATION)/$(ARCHIVE) $(DEPS)
+	rm -rf $(PLATFORM)/*; mkdir -p $(PLATFORM)
+	cd $(PLATFORM); $(ARCHIVE_TOOL) $(ARCHIVE_TOOL_FLAGS) $(TARBALLS_LOCATION)/$(ARCHIVE)
+
+.installed-$(PLATFORM): $(PLATFORM)
+	cd $(PLATFORM); $(NATIVEPREFIX)/bin/python3 -m pip install git+https://github.com/benfogle/crossenv.git
```

Comment:
I'm not sure why you unpack the archive then use pip to install a separate version?

Reply:
Ended up working this way. Realistically for our stuff I don't want to use pip, but this got it functioning first, so just stuck with it. I'll look to build instead of using pip

Reply:
Done, now build instead of using pip